src/actions/editor.js

```diff
@@ -89,15 +90,25 @@ export function setLanguages(languages) {
  * Replace all editor values
  * @param  {obj} formValues     new form values to replace all existing values
  */
-export function replaceData(formValues) {
-    return (dispatch) => {
-    // Run validations
-        dispatch(validateFor(null))
-        dispatch(setValidationErrors({}))
+export function replaceData(formValues, publicationStatus) {
```

Comment:
So we don't need the publicationStatus argument here.

Reply:
fixed in a new forced push. Default status to `public`